t/helpers.t

```diff
@@ -240,15 +235,15 @@ subtest 'set' => sub {
 
     subtest 'set date field with "" is an error' => sub {
```

Comment:
This test name needs to be updated to mention it's testing `email` now, and its goal is to test `format`.

Reply:
Quite right, done.